psiturk/api/__init__.py

```diff
@@ -278,6 +278,15 @@ def post(self):
         raise APIException(message='task name `{}` not recognized!'.format(data['name']))
 
 
+class BonusList(Resource):
+    def post(self):
```

Comment:
Should this be GET, not POST? my understanding of rest apis is that post is reserved for creation of new records, and GET for retrieval.

Reply:
I thought about this for a bit, and it's very possible to make it into a GET request. However, we'd then be unable to query for multiple assignment IDs. 

Actually, now that I think about it, I guess that that fits better with the MTurk boto3 client functionality itself. We'd just have to remove the multiple assignment ID workflows altogether. Will do when i find the time.